[EDU-4521] refactor: add tabs to Marketplace Integrations guides (#1172)
aziontech/docs: src/content/docs/en/pages/guides/marketplace/integrations/request-variation.mdx

Proceed as follows:

<Tabs client:visible>
    <Fragment slot="tab.console">Console</Fragment>
    <Fragment slot="tab.rtm">Real-Time Manager</Fragment>

<Fragment slot="panel.console">
1. On the upper-left corner, select **Products menu** > **Edge Application** on the **BUILD** section.
2. On the listing page of your edge applications, select the one you've created to use with the **Request Variation Controller** integration.
3. On the **Main Settings** tab, locate the **Edge Functions** switch and turn it on to enable Functions on your edge application.
4. Then, turn the **Application Accelerator** switch on to enable the *Forward Cookies* functionality for your application.
    - This will be addressed at the [Rules Engine section](#configuring-a-rule-on-rules-engine).
5. Click the **Save** button.

> You'll receive a message indicating your edge application was successfully updated.

Author: ```suggestion
You'll receive a success message indicating that your edge application has been updated.
```

</Fragment>

<Fragment slot="panel.rtm">
1. On the upper-left corner, select **Products menu** > **Edge Application** on the **BUILD** section.
2. On the listing page of your edge applications, select the one you've created to use with the **Request Variation Controller** integration.
3. On the **Main Settings** tab, locate the **Edge Functions** switch and turn it on to enable Functions on your edge application.
4. Then, turn the **Application Accelerator** switch on to enable the *Forward Cookies* functionality for your application.
    - This will be addressed at the [Rules Engine section](#configuring-a-rule-on-rules-engine).
5. Click the **Save** button.

> You'll receive a message indicating your edge application was successfully updated.

Author: ```suggestion
You'll receive a success message indicating that your edge application has been updated.
```
